src/plugins/dropdown_buttons/plugin.js

```diff
@@ -0,0 +1,39 @@
+Selectize.define('dropdown_buttons', function () {
```

Comment:
Can we move this plugin source code to our codebase? We are unlikely to merge this upstream, I don't think we want this change to be in the fork